src/som/interpreter/actors/Actor.java

```diff
@@ -182,6 +275,11 @@ protected void logMessageAddedToMailbox(final EventualMessage msg) { }
   protected void logMessageBeingExecuted(final EventualMessage msg) { }
   protected void logNoTaskForActor() { }
   public long getActorId() { return 0; }
+  public long getReplayActorId() { return 0; }
```

Comment:
Now I see the `ReplayActor`, or that one of course. Doesn't have to be the `TracingActor`, but, ideally only code that's really needed for all types of actors should be in the main class.